README.md

```diff
@@ -17,7 +17,7 @@ Please refer to the [Contribution guideline](.github/CONTRIBUTING.md) to contrib
 ## Nemo prerequisites and setup
 
 ### Prerequisites
-* Java 8
+* Java 11
```

Comment:
Does Java 8 still work?
Maybe something like this, so 8/9/10 users first give it a try?:

Java 8 or higher (mostly tested on Java 11)

Reply:
Sure, it's done